docs/tutorials/articles/understanding_gui/step_07/step_07.md

```diff
@@ -88,15 +88,15 @@ For example, this code creates a menu with two options:
                     lov=[('Page-1', 'Page 1'), ('Page-2', 'Page 2')],
                     on_action=menu_option_selected)
 
-            with tgb.Page() as page_1:
-                tgb.text("## This is page 1", mode="md")
-            with tgb.Page() as page_2:
-                tgb.text("## This is page 2", mode="md")
+        with tgb.Page() as page_1:
+            tgb.text("## This is page 1", mode="md")
+        with tgb.Page() as page_2:
+            tgb.text("## This is page 2", mode="md")
 
             pages = {
                 "/": root_page,
-                "page1": page_1,
-                "page2": page_2
+                "Page-1": page_1,
+                "Page-2": page_2
             }
             Gui(pages=pages).run()
```

Comment:
This is functional, but you should remove an indent so that `pages` and `Gui` are not a part of `page_2`.

Reply:
@AlexandreSajus Happy Holidays to you too!

Reply:
@AlexandreSajus Happy holidays to you too!